src/components/createEvent.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import '../css files/cards.css'
+
+class CreateEvent extends React.Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            name: '',
+            description: '',
+            startDate: '',
+            endDate: ''
+        }
+        this.handleChange = this.handleChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    handleChange(event) {
+        let nam = event.target.name
+        let val = event.target.value
+
+        this.setState({
+            [nam]: val
+        })
```

Comment:
There is an opportunity here to simplify this a bit with some destructuring:
` let {name, value} = event.target;
this.setState({name:value});`